ui/lib/pki/addon/templates/overview.hbs

```diff
@@ -19,7 +19,49 @@
 </Toolbar>
 
 {{#if this.model.hasConfig}}
-  {{! TODO show overview items }}
+  <div
+    class="selectable-card-container has-grid has-top-margin-l has-two-col-grid
+      {{if (or (eq this.model.issuers 403) (eq this.model.roles 403)) 'has-three-col-grid' 'has-two-col-grid'}}"
+  >
+    {{#if this.issuersCanList}}
```

Comment:
Pending ☝️ this conditional could be removed?

Reply:
@hellobontempo just confirmed with cryto, and the issuers endpoint is unauth-ed so that is why there's no permission error!